### PR TITLE
Update documentation/installation (zh_cn)

### PR DESCRIPTION
**Pull request by tzwm**
Add documentation/installation `index.md`.

@ruby/www-ruby-lang-org-i18n-zh_cn please help me review.

Note: it's part of #1194

### CODE REVIEW

**Comment by tzwm** [comment]
screen shot 2015-11-22 at 1 21 57 am

File: zh_cn/documentation/installation/index.md [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+
+[Uru][uru] 是一个轻量的、支持多平台的命令行工具，
+能够帮助你在 OS X、Linux 和 Windows 上使用不同版本的 Ruby。
+
+
+## 通过源码来编译安装
+{: #building-from-source}
+
+当然，你也可以通过源码来安装 Ruby。
+[下载](/en/downloads/) 、解压 tarball，然后执行：
```
Comment (hlcfan) [inline zh_cn/documentation/installation/index.md]: `[下载](/en/downloads/) ，解压 tarball，然后执行：` 
comma looks better?

File: zh_cn/documentation/installation/index.md [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+{: #rbenv}
+
+[rbenv][rbenv] 能让你管理在系统中安装的多个 Ruby。
+它不能直接用来安装 Ruby，
+但有个流行的插件叫 [ruby-build](#ruby-build) 能够用来安装 Ruby。
+
+这两个工具都支持 OS X、Linux 和其它类 UNIX 操作系统。
+
+
+### RVM（"Ruby Version Manager"）
```
Comment (hlcfan) [inline zh_cn/documentation/installation/index.md]: "" => “”

**Comment by JuanitoFatas** [comment]
LGTM.

**Comment by hlcfan** [comment]
:+1:

**Pull request merged**